app/views/hyrax/dashboard/sidebar/_repository_content.html.erb

```diff
@@ -0,0 +1,15 @@
+<li class="h5"><%= t('hyrax.admin.sidebar.repository_objects') %></li>
```

Comment:
For future reference when we're upgrading, please add some comments when doing an override of a hyrax file. For example: https://github.com/UCSCLibrary/ucsc-library-digital-collections/blob/master/app/views/hyrax/collections/_collection_description.html.erb

Reply:
Done